template.tpl

```diff
@@ -32,83 +32,227 @@ ___INFO___
       },
       {
-        "displayValue": "identify",
+        "displayValue": "Set User Properties (identify)",
         "value": "identify"
       },
       {
-        "displayValue": "setGroup",
+        "displayValue": "Set User ID (setUserId)",
+        "value": "setUserId"
+      },
+      {
+        "displayValue": "Set Group (setGroup)",
         "value": "setGroup"
       },
       {
-        "displayValue": "groupIdentify",
+        "displayValue": "Set Group Properties (groupIdentify)",
         "value": "groupIdentify"
       },
       {
-        "displayValue": "revenue",
+        "displayValue": "Track Revenue (revenue)",
         "value": "revenue"
       },
       {
-        "displayValue": "flush",
+        "displayValue": "Flush Events (flush)",
         "value": "flush"
       },
       {
-        "displayValue": "setUserId",
-        "value": "setUserId"
-      },
-      {
-        "displayValue": "setDeviceId",
+        "displayValue": "Set Device ID (setDeviceId)",
         "value": "setDeviceId"
       },
       {
-        "displayValue": "setSessionId",
+        "displayValue": "Set Session ID (setSessionId)",
         "value": "setSessionId"
       },
       {
-        "displayValue": "reset",
+        "displayValue": "Reset User (reset)",
         "value": "reset"
       },
       {
-        "displayValue": "setOptOut",
+        "displayValue": "Set Opt-Out (setOptOut)",
         "value": "setOptOut"
       }
     ],
     "displayName": "Tag Type",
     "defaultValue": "init",
     "simpleValueType": true,
     "name": "type",
-    "type": "SELECT"
+    "type": "SELECT",
+    "alwaysInSummary": true
+  },
+  {
+    "type": "LABEL",
+    "name": "initDescription",
+    "displayName": "\u003ca href\u003d\"https://www.docs.developers.amplitude.com/data/sources/google-tag-manager-client/#init\"\u003einit\u003c/a\u003e - This tag sets up Amplitude on the page. You likely only need one init tag, and it should be triggered on All Pages or a similar trigger that fires consistently on each page load. See also \u003ca href\u003d\"https://www.docs.developers.amplitude.com/data/sdks/browser-2/#initialize-the-sdk\"\u003eSDK Reference\u003c/a\u003e",
+    "enablingConditions": [
+      {
+        "paramName": "type",
+        "paramValue": "init",
+        "type": "EQUALS"
+      }
+    ]
+  },
+  {
+    "type": "LABEL",
+    "name": "trackDescription",
+    "displayName": "\u003ca href\u003d\"https://www.docs.developers.amplitude.com/data/sources/google-tag-manager-client/#track\"\u003etrack\u003c/a\u003e - This tag sends an \u003ca href\u003d\"https://www.docs.developers.amplitude.com/analytics/what-is-amplitude/#events\"\u003eevent\u003c/a\u003e to Amplitude. Events are actions that users take in your product, such as clicking a button, making a purchase, or signing up for an account. See also \u003ca href\u003d\"https://www.docs.developers.amplitude.com/data/sdks/browser-2/#tracking-an-event\"\u003eSDK Reference\u003c/a\u003e",
```

Comment:
i think the GTM-specific dev docs page should be sufficient but if not we should improve them. We can keep the SDK link but should point out that it's a highly technical doc.

```suggestion
    "displayName": "\u003ca href\u003d\"https://www.docs.developers.amplitude.com/data/sources/google-tag-manager-client/#track\"\u003etrack\u003c/a\u003e - This tag sends an \u003ca href\u003d\"https://www.docs.developers.amplitude.com/analytics/what-is-amplitude/#events\"\u003eevent\u003c/a\u003e to Amplitude. Events are actions that users take in your product, such as clicking a button, making a purchase, or signing up for an account. Refer to \u003ca href\u003d\"https://www.docs.developers.amplitude.com/data/sdks/browser-2/#tracking-an-event\"\u003eSDK Documentation\u003c/a\u003e for technical details.",
```

Reply:
Thinking more about it, I would love to see this GTM template and its docs not rely on Browser SDK for information/documentation/references because the docs around the SDK are highly technical and may not be relatable to customers instrumenting with GTM. While they perform the same function and are convenient to link with each other, customers shouldn't need to know that

Reply:
Ok I'll remove the sdk doc links

Reply:
Ideally, we should link all to the gtm specific docs. However, It's like a chicken-egg problem. The gtm template and the doc depend on each other. I think we can first link it to the browser sdk doc, and update the doc reference to link to the gtm doc later.